### PR TITLE
eslint 5.4.0 (new formula)

### PR DESCRIPTION
**Pull request by tommyang**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/eslint.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    (testpath/".eslintrc.json").write("{}") # minimal config
+    (testpath/"syntax-error.js").write("{}}")
+    # https://eslint.org/docs/user-guide/command-line-interface#exit-codes
+    shell_output("#{bin}/eslint syntax-error.js", 1)
```
Comment (DomT4) [inline Formula/eslint.rb]: Can we `assert_match` on the `Unexpected token` message this throws?

**Review by DomT4 — DISMISSED** [review]

File: Formula/eslint.rb (hunk 1 above)
Comment (tommyang) [reply Formula/eslint.rb]: I guess we can. However, I based this test on [ESLint's own test](https://github.com/eslint/eslint/blob/1e88170fa57656206a39d579fd0c176d414c275b/tests/lib/cli.js#L270-L277) which only checks the exit code.

Comment (DomT4) [reply Formula/eslint.rb]: It's just insurance to make sure we're always getting the expected failure.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/8c1c23d80cf33391f2419ab73e3e1a42b8afa293. Thanks again @tommyang!

**Pull request closed without merging**